sonaranalyzer-dotnet/src/SonarAnalyzer.RuleDescriptorGenerator/packages.lock.json

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
The packages.lock.json files were generated to enable caching on azure pipelines